modules/terraform-before-cluster.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * 
+
+:_content-type: PROCEDURE
+
+[id="terraform-before-cluster"]
+= Before creating a cluster
+The following use cases provide examples of actions taken before cluster creation.
+
+== Terraform prerequisites
+//modules/sd-terraform-prereqs.adoc
+//undo hiding the above include when Eric *finally* pushes 7235
+[id="terraform-iam-roles"]
+== Account-wide IAM roles
+Prior to creating a ROSA STS cluster, you must create the required account-wide roles and policies. See "Account-wide IAM role and policy reference" in the Additional resources for more information.
+
+.Prerequisites
+* You have an AWS account.
+* You have installed the latest version of the ROSA command-line (CLI) (`rosa`).
+* You have an offline OpenShift Cluster Manager token. This token can be generated in the Red Hat Hybrid Cloud Console link:https://console.redhat.com/[Red Hat Hybrid Cloud Console].
+* You have installed Terraform. See the link:https://developer.hashicorp.com/terraform/downloads[Terraform page]for the latest version.
+
+//.Open issue
+//This example also creates the policies for the Operator roles. When you update the version, it also updates the Operator role policies.
+
+.Procedure
+
+. To run `terraform apply` you need to set up some variables. This guide uses environmental variables. For more on Terraform variables, see link:https://developer.hashicorp.com/terraform/enterprise/workspaces/variables/managing-variables[Managing Variables] in the Terraform documentation.
```

Comment:
Let's revise it to something like "When running Terraform, you configure your infrastructure through variables. See Hashicorp docs for information on using variables in Terraform. The following variables need to be defined to create your resources." Then we just list the resources from there.

Thoughts?